### PR TITLE
Code Review Request.

### PR DESCRIPTION
**Pull request by KelynPNjeri**
Herein is the code for the Restaurant Page application. It makes use of Vanilla JS and Webpack to transpile the Javascript and manage and optimize the assets within the project.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "Simple Restaurant Page",
+  "private": "true",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "watch": "webpack --watch"
+  },
```
Comment (Dandush03) [inline package.json:10]: ### Scripts:
### Let's add the proper inviroment for each script
```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const path = require('path');
+
```
Comment (Dandush03) [inline webpack.config.js:2]: ### webpack
- [ ]  usage of `import` and `export` instead of `require` or `module.exports`

**Review by Dandush03 — CHANGES_REQUESTED** [review]
# Status: :red_circle: Require Changes :red_circle: 
### Great work so far :muscle: checkout my comments below
- [x] It features a tabbed menu. When a tab is clicked the content is replaced ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/tab_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page))
- [x] The contents of each tab are stored in a `module`. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/tab_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page))
- [ ] Each `module` exports a function that creates a `<div>` element, adds the appropriate content and styles to that element and then appends it to the DOM. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/tab_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page))

# Happy Coding :computer: :nerd_face:

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required  🏗️ 🔨 
****
### Hello @KelynPNjeri   👋 
You are doing a great job with this project! 👍 You are almost done! 💪 Great job applying the feedback you have received from the previous TSE. 🎖️ 🌟 There are just a few small issues that need to be resolved before you can move on: ⬇️ 
****

- [ ] When the user clicks on a tab that they are already, it adds the information to the page another time. See my screenshots below for clarification 👇🏾 
Screenshot from first time user clicks 'contact us' 👇🏾 
Screen Shot 2021-01-18 at 3 19 26 PM
Screenshot when user clicks contact us again 👇🏾 
Screen Shot 2021-01-18 at 3 19 49 PM
* This happens on both the contact us tab and the main menu tab. 


****
### Kindly make the required changes and resubmit for a new code review once you are ready 🙏


Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hi @KelynPNjeri :hand: 
Great job implementing the requested changes.
However, the linter checks didn't pass in your last commit. 

- [x] We need to make sure both eslint and stylint checks pass

Happy coding :computer: 
**Sergio Torres**

**Review by Torres-ssf — APPROVED** [review]
### PR Approved :+1: 
Hello again.
Your PR is approved.
You may merge the PR now.

Happy coding :computer: 
**Sergio Torres**

**Pull request merged**